src/components/Map/OperatorSidebarParcelsSubmit.vue

```diff
@@ -1,64 +1,76 @@
 <template>
-  <div class="py-3">
-    <section class="px-3">
-      <p class="d-flex red--text text--darken-3 font-weight-bold" v-if="isSupposedToHaveParcels">
-        <v-icon color="red darken-3" class="mr-3">warning</v-icon>
-        Désolé, nous n'avons pas trouvé de parcelles en {{ baseYear }}
-        pour cet opérateur.
-      </p>
-      <p class="d-flex font-weight-bold" v-else-if="operator.numeroPacage">
-        <v-icon class="mr-3">help_outline</v-icon>
-        Les parcelles de cet opérateur ne nous ont pas été communiquées.
-      </p>
+  <v-form class="px-3" v-model="isFormValid">
+    <h2 class="subheading text--cyan">Ajouter une parcelle</h2>
 
-      <v-subheader class="text--cyan">Renseigner le parcellaire</v-subheader>
+    <h3 class="body-2 mt-2">1. Références cadastrales</h3>
 
-      <p class="body-1 grey--text text--darken-2">
-        Renseignez le parcellaire de l'opérateur&nbsp;bio n°{{ operator.numeroBio }}
-        ({{ operator.title }}), et nous les intégrerons à CartoBio.
+    <p class="body-1" v-if="!hasCadastralReferences">
+      Sélectionnez une ou plusieurs parcelles cadastrales sur la carte.
+    </p>
+    <p class="body-1 grey--text text--darken-2" v-else>
+      Continuez à sélectionner des sections cadastrales, ou renseignez les informations de culture.
+    </p>
+
+    <ul class="parcels-list pl-3 py-2" v-if="hasCadastralReferences">
+      <li v-for="feature in parcel.cadastralReferences" :key="feature.id">
+        Section&nbsp;<strong>{{ feature.properties.section }}</strong>, parcelle&nbsp;<strong>{{ feature.properties.numero }}</strong>
+        <span class="pl-1 grey--text text--darken-1">({{ beautifyNumber(feature.properties.contenance / 10000) }}ha)</span>
+      </li>
+    </ul>
+
+    <section>
+      <h3 class="body-2 mt-2">2. Culture</h3>
+
+      <p class="body-1 grey--text text--darken-2" v-if="!hasCadastralReferences">
+        <v-icon small color="grey--text text--darken-2">info</v-icon>
+        Les champs suivants seront modifiables après sélection d'au moins une référence cadastrale.
       </p>
-    </section>
 
-    <v-expansion-panel elevation-0>
-      <v-expansion-panel-content>
-        <template v-slot:header>
-          <h3 class="subheading">Saisie libre</h3>
-        </template>
+      <v-autocomplete :rules="[rules.required]" :disabled="!hasCadastralReferences" dense :items="knownCultures" label="Type de culture" prepend-icon="search" item-text="Libellé Culture" item-value="Code Culture" v-model="parcel.culturalCode" />
+      <v-select :rules="[rules.required]" :disabled="!hasCadastralReferences" :items="conversionStatuses" prepend-icon="list" label="Statut de la conversion" v-model="parcel.conversionStatus" />
+      <v-text-field :disabled="!hasCadastralReferences" label="Année de début de conversion" prepend-icon="event" v-model="parcel.conversionDate" />
```

Comment:
La date d'engagement des parcelles est enregistrée par les OC et peut avoir son importance vs juste l'année, si l'engagement est récent.
Peut être avoir un champ année + un autre facultatif 'date' ?

Reply:
Oui ça peut le faire. Ça sera à itérer de toutes façons, pour comprendre quels champs les intéressent, comment ça marche avec une rotation de culture bi-annuelle, et si ça serait pas plus clean de demander l'historique connu.